api/metrics/urls.py

```diff
@@ -7,4 +7,5 @@
 urlpatterns = [
     url(r'^preprints/views/$', views.PreprintViewMetrics.as_view(), name=views.PreprintViewMetrics.view_name),
     url(r'^preprints/downloads/$', views.PreprintDownloadMetrics.as_view(), name=views.PreprintDownloadMetrics.view_name),
+    url(r'^institutions/$', views.InstitutionSummaryMetrics.as_view(), name=views.InstitutionSummaryMetrics.view_name),
```

Comment:
@brianjgeiger Is this metrics or institutions endpoint? These are metrics views, but are I made the department endpoint [off institutions](https://github.com/CenterForOpenScience/osf.io/pull/9318/files#diff-e8a52b87015f3732c0abf6916bc414d4R16)

Reply:
I think they should be institutions endpoints. They'll need to be fully JSON-API compliant and used by our front-end, so no need to put them in the `/_/metrics` namespace.